src/features/blog/containers/newsContainer/index.test.js

```diff
@@ -1 +1,71 @@
-xtest('test placholder', () => {})
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import FilterButton from '../../../../components/filterButton'
+import NewsCard from '../../components/newsCard'
+import NewsContainer from '.'
+
+const makeTestArticles = (quantity, category) => {
+  const array = []
+  for (let i = 0; i < quantity; i++) {
+    array.push({
+      id: `${i}-${category}`,
+      slug: 'news-and-views/title',
+      title: 'Title',
+      readTime: null,
+      datePublished: '2019-12-19',
+      category: { title: `${category}`, hexColour: '#2CDA9D' },
+    })
+  }
+  return array
+}
+
+const testArticles = [
+  ...makeTestArticles(2, 'Research'),
+  ...makeTestArticles(9, 'General'),
+]
+const testCategories = [
+  { title: 'All Articles', hexColour: '#FFD95E' },
+  { title: 'Research', hexColour: '#2CDA9D' },
+]
+
+describe(NewsContainer.name, () => {
+  beforeAll(() => {
+    global.___loader = {
```

Comment:
As suggested in the [Gatsby docs](https://www.gatsbyjs.org/docs/unit-testing/#2-creating-a-configuration-file-for-jest) this should go into the setup file for Jest. In our codebase, this is the `config/jest.setup.js` file.

Reply:
Several unit tests also teardown the ___loader. Would if be appropriate to add `resetMocks: true` to the Jest config? There are few options but not sure what will interact best with other tests.
```
  afterAll(() => {
    global.___loader.enqueue.mockReset()
  })
```

Reply:
We shouldn't need this 🙂